core/src/main/java/org/mule/api/util/CredentialsMaskUtil.java

```diff
@@ -79,22 +84,38 @@ public static String maskUrlUserAndPassword(String input, Pattern passwordPatter
     }
 
     /**
-     * masks password attribute
+     * Masks password attribute
      * 
      * @param password password to be masked
-     * 
      * @return password masked
      */
     public static String maskPasswordAttribute(String password)
     {
         return format(PASSWORD_ATTRIBUTE_MASK, password);
     }
 
-    private static String maskUrlPattern(String input, Pattern pattern, String mask) {
+    /**
+     * Masks pattern in input
+     * 
+     * @param input credentials for user and password to be masked
```

Comment:
Again, input is not a meaningful parameter name. Rename it.
Also, this method can be used to mask anything, so user and password as only examples of waht can be masked. Fix the javadoc